### ISSUE TITLE
Project_sla issue

### ISSUE THREAD

**Issue by aitorbouzas** [OCA/project#70]
Hi,

Just installed the project_sla in Odoo 8 but I can't see things working. In your setup checklist from the **openerp**.py file I can see that "Working Calendar" must be set into Project's "Other Info" tab. Where is that "Working Calendar" or what do you mean? I don't see it...

When I create a contract, with it's project and I link a SLA to it, every issue I try to create in the project just keeps the server loading. It's like everything related with a SLA won't work.

P.D: had an error in project_sla.py with 'control_field_id' domain. ('type','in',['date','datetime']) did not work, for my instance (don't know why) 'type' field was named 'ttype' in ir.model.fields model.

Thanks

**Comment by dreispt** [comment]
From what I can see, the code is using `ttype` not `type`: see https://github.com/OCA/project-service/blob/8.0/project_sla/project_sla.py#L37
What Odoo and module versions are you using?

The "Working calendar" should be visible, we need to fix that.
As a work around, you can make it visible by enabling the "Manage time estimation on tasks" option on Settings -> Configuration Project. 
After that you should see a "Working Time" selection field on the Project "Other Info" tab.
It's just a visibility setting, so you can later disable it without losing the information.

**Comment by aitorbouzas** [comment]
I don't know why my code wasn't using 'ttype'... Maybe I downloaded an old version, I just did a 'git clone'.

Odoo is 8.0, and your module I thought it was the last one, but it seems that i was wrong. 

The workaround was nice, now I can set a Working Time, but after clicking the "Reapply SLA" button nothing appeared in Project Issue. This is my SLA configuration for the contract:

![captura](https://cloud.githubusercontent.com/assets/11627953/7043507/4d20c1cc-ddea-11e4-872b-9196831b816e.PNG)

And this is the definition:

![captura](https://cloud.githubusercontent.com/assets/11627953/7043515/6321e9a6-ddea-11e4-9ed6-8e9552da3c7e.PNG)

Then, this is the working time:

![captura](https://cloud.githubusercontent.com/assets/11627953/7043536/798226f2-ddea-11e4-8eb6-093131cc8858.PNG)

Maybe something is missing? That working time, of course, is assigned to a project that is related to the contract that has the previous SLA definitions.

Thank you very much for your response @dreispt !
